doc/charm++/manual.rst

```diff
@@ -10976,6 +10976,34 @@ superclass to do the final delivery after you’ve sent your messages.
 Experimental Features
 =====================
 
+CMI-SHMEM
```

Comment:
I think it should be called "Shmem" here and anywhere a user might see it. Charm++ users shouldn't need to know anything about Converse and its Cmi namespace.